tests/models/vit_mae/test_modeling_vit_mae.py

```diff
@@ -254,6 +254,35 @@ def test_save_load(self):
                 max_diff = np.amax(np.abs(out_1 - out_2))
                 self.assertLessEqual(max_diff, 1e-5)
 
+    def test_initialization(self):
+        config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
+
+        configs_no_init = _config_zero_init(config)
+        for model_class in self.all_model_classes:
+            model = model_class(config=configs_no_init)
+            for name, param in model.named_parameters():
+                uniform_init_parms = [
```

Comment:
I'd move this up out of the for-loop as it doesn't need to be redefined for each parameter